python/dlisio/lis/load.py

```diff
@@ -1,17 +1,15 @@
 import logging
 
 from .. import core
-from .file import logical_file, physical_file
+from .file import logical_file, physical_file, parse_record
 
 def load(path):
-    """ Loads and indexes a file
+    """ Loads and indexes a LIS file
 
-    A LIS file is typically segmented into multiple Logical Files (LF). LF's
-    are just a way to group information that logically belongs together. LF's
-    are completely independent of each other and can generally be treated as if
-    they were different files on disk. In fact - that's just what load does.
-    Each logical file gets its own io-device and is completely segmented from
-    other LF's.
+    Load does more than just opening the file. A LIS file have no random access
```

Comment:
1. 👍 
2. Not directly probably. But the fact that load scans the entire file might be. That is quite unexpected from a "load" function (although we are used to it with dlis). So that knowledge might set the right expectations wrt performance. And potential errors might be less surprising if you understand that load needs to scan the file from start to finish